### PR TITLE
SLM-194: Introduced artillery inc. html report output

### PR DESCRIPTION
**Pull request by nathanrussell-moj-digital**

### CODE REVIEW

File: artillery/artillery [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+echo Starting SLM API and supporting containers via docker-compose
+docker-compose up -d
```
Comment (mikehalmamoj) [inline artillery/artillery:4]: Worth sticking a `pull` in here? Or at least mentioning that in the readme?

File: artillery/artillery.yml [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    - duration: 10
+      arrivalRate: 1
+      name: Warm Up
+    - duration: 30
+      arrivalRate: 1
+      rampTo: 20
+      name: Ramp Up
+    - duration: 300
+      arrivalRate: 20
+      name: Sustained Load
```
Comment (mikehalmamoj) [inline artillery/artillery.yml:19]: I guess this is the configuration I might want to play with - might be nice to mention this in the readme? And maybe point to the Artillery documentation of this?

File: artillery/.gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+test-run-report.json
+test-run-report.json.html
```
Comment (mikehalmamoj) [inline artillery/.gitignore:2]: Does this need the .env file too?

Comment (nathanrussell-moj-digital) [reply artillery/.gitignore:2]: No, the env file has been deleted in the latest commit as its not used anymore

Comment (mikehalmamoj) [reply artillery/.gitignore:2]: Ah I thought I'd deleted it before running this version... guess not!

**Review by mikehalmamoj — APPROVED** [review]

File: artillery/artillery [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+      RAMP_UP_TARGET_VUSERS=$2
+      shift # past argument
+      shift # past value
+      ;;
+    --sustained-load-vusers)
+      SUSTAINED_LOAD_VUSERS=$2
+      shift # past argument
+      shift # past value
+      ;;
+  esac
```
Comment (mikehalmamoj) [inline artillery/artillery:48]: In a future PR might be nice to have a default *) here to catch misspelled parameters and bail out.

**Pull request merged**